notebooks/tel_and_site/subsys_req_ver/m2/SITCOM2102_M2Hexapod_fault_study.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>Recommendation - consider using <a href="https://sembr.org/" target="_blank">Semantic Line breaks</a>. </p><p>They improve readability in this kind of documentation. </p><p>However, feel free to ignore this recommendation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>